### PR TITLE
Basic splash screen implemented

### PR DESCRIPTION
**Pull request by Codeek89**
PS. I had to change the sdk version since vsync parameter in the "welcome_screen" animation couldn't be found by android studio.

### CODE REVIEW

File: lib/presentation/screens/splash_screen.dart [hunk 1]
```diff
@@ -0,0 +1,34 @@
...
+  static const String id = "splash_screen";
+  @override
+  _PunkSplashScreenState createState() => _PunkSplashScreenState();
+}
+
+class _PunkSplashScreenState extends State<PunkSplashScreen> {
+  @override
+  void initState() {
+    super.initState();
+    Timer(Duration(seconds: 2),
```
Comment (hanskokx) [inline lib/presentation/screens/splash_screen.dart:16]: Is this an arbitrary value? It should only show the splash screen as long as the application is loading.

File: lib/presentation/screens/splash_screen.dart [hunk 2]
```diff
@@ -0,0 +1,34 @@
...
+        () => Navigator.pushReplacementNamed(context, WelcomeScreen.id));
+  }
+
+  @override
+  Widget build(BuildContext context) {
+    return Container(
+      color: Colors.white,
+      child: Center(
+        child: Container(
```
Comment (hanskokx) [inline lib/presentation/screens/splash_screen.dart:25]: Container in a container? Why not Expanded()?

**Review by hanskokx — CHANGES_REQUESTED** [review]

File: lib/presentation/screens/splash_screen.dart (hunk 2 above)
Comment (Codeek89) [reply lib/presentation/screens/splash_screen.dart:25]: you're right I didn't think about it

File: lib/presentation/screens/splash_screen.dart (hunk 1 above)
Comment (Rift3000) [reply lib/presentation/screens/splash_screen.dart:16]: I agree

File: lib/presentation/screens/splash_screen.dart (hunk 2 above)
Comment (Rift3000) [reply lib/presentation/screens/splash_screen.dart:25]: Looking forward to your next PR

File: lib/presentation/screens/splash_screen.dart (hunk 1 above)
Comment (Codeek89) [reply lib/presentation/screens/splash_screen.dart:16]: okok, so I have to work with xml, for what concerns the android part.

**Comment by Codeek89** [comment]
I made a new commit

**Review by hanskokx — APPROVED** [review]

**Review by Rift3000 — APPROVED** [review]

**Comment by Rift3000** [comment]
Nicely done

**Comment by Codeek89** [comment]
Thank youuu

**Pull request merged**